src/extensions/default/InlineColorEditor/InlineColorEditor.js

```diff
@@ -124,12 +114,12 @@ define(function (require, exports, module) {
         // the matched length here.
         if (matches && (end.ch === undefined || end.ch - start.ch < matches[0].length)) {
             end.ch = start.ch + matches[0].length;
-            this._endBookmark.clear();
-            this._endBookmark = this.hostEditor._codeMirror.setBookmark(end);
+            this._marker.clear();
+            this._marker = this.hostEditor._codeMirror.markText(start, end);
```

Comment:
I did that and found that there's a lot of shared code in `getCurrentRange` - where to put shared utility functions for Inline Editor Extensions?